scalableminds/webknossos
# Refactor Backend: Merge ScriptSQL and Script. Also Analytics

### URL of deployed dev instance (used for testing):
- https://refactorbackendscript.webknossos.xyz

### Steps to test:
- create scripts, delete, edit
- create task with and without script
- script should be loaded when working on the respective task

### Issues:
- contributes to #2521

------
